src/parenttext/canto.py

```diff
@@ -62,15 +63,15 @@ def authorize(self):
         )
 
     def tree(self, folder_id: str):
-        return (
-            requests.request(
-                method="GET",
-                url=f"{self.site_base_url}/api/v1/tree/{folder_id}",
-                headers={"Authorization": f"Bearer {self.token}"},
-            )
-            .json()
-            .get("results", [])
+        req = requests.request(
+            method="GET",
+            url=f"{self.site_base_url}/api/v1/tree/{folder_id}",
+            headers={"Authorization": f"Bearer {self.token}"},
         )
+        if req.status_code == 401:
+            print("401 Unauthorized: Ensure environment variables are loaded")
+            raise
```

Comment:
I think there is no active exception to re-raise here, so this will probably raise a `RuntimeError` that won't have anything to do with the real cause (authorisation failure). I would replace the `print` statement with `raise Exception` with the same or similar message as the `print` statement.